tsconfig.json

```diff
@@ -1,24 +1,17 @@
+/*
+  This is a "Solution Style" tsconfig.json file, and is used by editors and TypeScript’s language server to improve development experience.
+  It is not intended to be used to perform a compilation.
+
+  To learn more about this file see: https://angular.io/config/solution-tsconfig.
+*/
 {
-  "compilerOptions": {
-    "baseUrl": "./",
-    "importHelpers": true,
-    "outDir": "./dist/out-tsc",
-    "strict": true,
-    "sourceMap": true,
-    "declaration": false,
-    "module": "esnext",
-    "target": "es5",
-    "moduleResolution": "node",
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "esModuleInterop": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "downlevelIteration": true,
-    "typeRoots": ["node_modules/@types"],
-    "lib": ["es2015", "es2016", "es2017", "esnext.array", "dom"],
-    "paths": {
-      "@hypertrace/hyperdash-angular": ["projects/hyperdash-angular/src/public_api.ts"]
+  "files": [],
+  "references": [
```

Comment:
I think this will fix all the IDE related mapping that we had to put in individual libraries